### PR TITLE
argobots: update to v1.2

### PR DESCRIPTION
**Pull request by yfguo**
Upgrading to latest Argobots release and making v1.2 the default.

### CODE REVIEW

**Review by carns — APPROVED** [review]

**Review by tldahlgren — DISMISSED** [review]
Confirmed the version sha256.

**Comment by tldahlgren** [comment]
@shintaro-iwasaki Do you want to comment on this simple change before the PR is merged?

**Comment by yfguo** [comment]
> @shintaro-iwasaki Do you want to comment on this simple change before the PR is merged?

Shintaro is no longer working on the Argobots project. I am the only maintainer now.

**Comment by tldahlgren** [comment]
> > @shintaro-iwasaki Do you want to comment on this simple change before the PR is merged?
> 
> Shintaro is no longer working on the Argobots project. I am the only maintainer now.

Ah.  Do you want to remove him as a maintainer from the package?  And add yourself as a maintainer?

**Comment by yfguo** [comment]
> > > @shintaro-iwasaki Do you want to comment on this simple change before the PR is merged?
> > 
> > 
> > Shintaro is no longer working on the Argobots project. I am the only maintainer now.
> 
> Ah. Do you want to remove him as a maintainer from the package? And add yourself as a maintainer?

Yes, please do that.

File: var/spack/repos/builtin/packages/argobots/package.py
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/argobots/package.py]:
```suggestion
    maintainers("yfguo")
```

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**
